internal/unit/file.go

```diff
@@ -98,7 +102,11 @@ func (u *File) Insert(section, name string, value ...string) *File {
 }
 
 // Overwrite overwrites name=value in the section and returns a new File.
+// If name is the empty string this is a noop.
 func (u *File) Overwrite(section, name string, value ...string) *File {
+	if name == "" {
```

Comment:
Please, add a comment above stating that this is empty only when the option is not supported by the current systemd and no alternative is available.